install/silverbullet-install.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: Dominik Siebel (dsiebel)
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y curl
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+msg_ok "Installed Dependencies"
+
+RELEASE=$(curl -s https://api.github.com/repos/silverbulletmd/silverbullet/releases/latest | grep "tag_name" | awk '{print substr($2, 2, length($2)-3) }')
+
+msg_info "Installing ${APPLICATION}"
+mkdir -p /opt/silverbullet/bin /opt/silverbullet/space
+wget -q https://github.com/silverbulletmd/silverbullet/releases/download/${RELEASE}/silverbullet-server-linux-x86_64.zip
+unzip silverbullet-server-linux-x86_64.zip &>/dev/null
+mv silverbullet /opt/silverbullet/bin/
+chmod +x /opt/silverbullet/bin/silverbullet
+ln -s /opt/silverbullet/bin/silverbullet /usr/local/bin/silverbullet
+echo "${RELEASE}" >/opt/silverbullet/${APPLICATION}_version.txt
+msg_ok "Installed ${APPLICATION}"
```

Comment:
APP

Reply:
See above, didn't work for me, only `$APPLICATION` was available in install scripts.